src/components/TeamCard.js

```diff
@@ -17,7 +17,7 @@ const TeamCard = ({ name, avatar, github }) => (
                 <LazyLoadImage alt={name} src={avatar} />
             </div>
             <div className="flex flex-col flex-none items-start">
-                <div className="-mt-8 text-2xl font-inter text-stone-600 dark:text-zinc-300 text-left">{name}</div>
+                <div className="-mt-8 text-2xl font-inter text-stone-600 dark:text-zinc-300 text-left overflow-hidden lg:text-xl">{name}</div>
```

Comment:
I was looking into trying to use ellipses when it overflowed but couldn't make it work, maybe you can?

Reply:
I tried, but using tailwind's native truncate property didn't seem to work fully. https://stackoverflow.com/questions/53497656/tailwind-text-overflow-ellipsis

<img width="927" alt="image" src="https://user-images.githubusercontent.com/51520568/210445592-61d2695f-aa05-4d3c-98a0-d7c788913eac.png">

It truncates fine, but I can't seem to get ellipses at all for some reason. If you'd like, I can commit this here without the ellipses?